pkgs/development/libraries/gdal/default.nix

```diff
@@ -103,6 +104,8 @@ stdenv.mkDerivation (finalAttrs: {
     "-DGDAL_USE_INTERNAL_LIBS=OFF"
     "-DGEOTIFF_INCLUDE_DIR=${lib.getDev libgeotiff}/include"
     "-DGEOTIFF_LIBRARY_RELEASE=${lib.getLib libgeotiff}/lib/libgeotiff${stdenv.hostPlatform.extensions.sharedLibrary}"
+    "-DLERC_INCLUDE_DIR=${lib.getDev lerc}/include" # TODO: GEOTIFF, GIF, JPEG, JSONC, LERC, OPENCAD, PNG, QHULL, TIFF, ZLIB
```

Comment:
Can you explain this TODO?

Reply:
This was for my benefit, but basically once LERC support has been solved I want to also make sure all those internal libraries (disabled by `-DGDAL_USE_INTERNAL_LIBS=OFF`) work.

Reply:
It's a reminder for me to make sure all the internal libraries are linked and tested.